config/config.py

```diff
@@ -7,19 +7,28 @@ def playerIcons(poi):
         return "Last known location for {}".format(poi['EntityId'])
 
 
-# Only render the signs with the filter string (stored in environment variable
-# RENDER_SIGNS_FILTER) in them. If RENDER_SIGNS_FILTER is blank or unset,
-# render all signs. RENDER_SIGNS_FILTER defaults to "-- RENDER --" for historic
-# reasons, but also so that people can have secret bases and to keep the render
-# fairly clean.
+# Only render the signs with the filter string in them. If filter string is
+# blank or unset, render all signs. Lines are joined with a configurable string.
 def signFilter(poi):
     # Because of how Overviewer reads this file, we must "import os" again here.
     import os
+    # Only render signs with this function
     if poi['id'] in ['Sign', 'minecraft:sign']:
-        sign_filter_string = os.environ.get('RENDER_SIGNS_FILTER', "")
-        render_all_signs = len(sign_filter_string) == 0
-        if render_all_signs or sign_filter_string in poi.values():
-            return "<br />".join([poi['Text1'], poi['Text2'], poi['Text3'], poi['Text4']])
+        sign_joiner = os.environ.get('RENDER_SIGNS_JOINER', "<br />")
+        sign_filter = os.environ.get('RENDER_SIGNS_FILTER', "")
+        hide_filter = os.environ.get('RENDER_SIGNS_HIDE_FILTER', 'false').lower() == 'true'
```

Comment:
Will any of the environment vars ever be unset? The Dockerfile sets them to a default already.

Reply:
Removed `.get` in f5952e7 and 38cbfe2.